taskk10.6/Program.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace taskk10._6
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.OutputEncoding = System.Text.Encoding.UTF8;
+
+            Console.WriteLine("Введите значение a: ");
+            int a = int.Parse(Console.ReadLine());
+
+            Console.WriteLine("Введите значение b (должно быть больше a): ");
+            int b = int.Parse(Console.ReadLine());
+
+            Console.WriteLine("Введите значение k (2 ≤ k ≤ 10): ");
+            int k = int.Parse(Console.ReadLine());
+
+            if (a >= b(k > 10))
```

Comment:
Ошибка в выражении. Пропущен знак операции дизъюнкция. Лишние скобки лучше не писать.
```suggestion
            if (a >= b || k > 10)
```